<?php
function inverse($x)
{
	if($x == 0)
	{
		throw new Exception('Division by zero.');
	}
	else
	{
		return 1 / $x;
	}
}

echo "start!\n";
try
{
	echo inverse(5) . "\n";
	echo inverse(0) . "\n";
	echo inverse(3) . "\n";
}
catch (Exception $e)
{
	echo "Caught exception: " . $e->getMessage() . "\n";
}
echo "end!\n";
